runtime/inspector/mod.rs

```diff
@@ -405,26 +55,12 @@ enum PollState {
 
 pub struct DenoInspector {
   v8_inspector_client: v8::inspector::V8InspectorClientBase,
-  v8_inspector: v8::UniquePtr<v8::inspector::V8Inspector>,
-  sessions: RefCell<InspectorSessions>,
+  // TODO(bartlomieju): make this field private
+  // TODO(bartlomieju): could probably be v8::UniqueRef instead of UniquePtr
+  pub v8_inspector: Rc<RefCell<v8::UniquePtr<v8::inspector::V8Inspector>>>,
+  sessions: RefCell<SessionContainer>,
   flags: RefCell<InspectorFlags>,
   waker: Arc<InspectorWaker>,
-  _canary_tx: oneshot::Sender<Never>,
```

Comment:
As per convo on Discord - not sure what this channel was used for - the only place it was referenced was inside `server.rs` to "deregister" the inspector instance from the server - however, given that `canary_tx` was never used, the reference was effectively dead and thus I removed it. 

I have a feeling it might have been something that was not finished, in which case I will revert this change.

CC @piscisaureus

Reply:
The purpose of `_canary_tx` is to let the websocket server thread detect when its counterpart `Inspector` object on the main thread is dropped (this tells the server thread that it should exit). No message is ever sent over this channel, however when the `send` side is dropped the `receive` side wakes up. It is used here: https://github.com/denoland/deno/blob/428bc6849f36ea31f32a42cb8acf64a5902659bf/runtime/inspector.rs#L260-L266

Reply:
Right, I thought that's the case; then my question is: couldn't this be rewritten to explicitly send a message over a channel in `Drop` implementation of the inspector? There was not a single comment regarding this canary channel and it's tx side was prefixed with underscore so it's quite hard to figure that out.

Reply:
Brought back as `deregister_tx`/`deregister_rx`